packages/collector/test/tracing/messaging/node-rdkafka/test.js

```diff
@@ -291,6 +302,154 @@ mochaSuiteFn('tracing/messaging/node-rdkafka', function () {
     ]);
   }
 
+  describe('tracing enabled, header format string', function () {
```

Comment:
We did not have tests with different header formats for rdkafka before. `binary` is not used, so it is fine that that is missing, but a test for `string` was missing so I added it.